src/material.cpp

```diff
@@ -64,36 +64,32 @@ namespace {
   Endgame<KPsK>   ScaleKPsK[]   = { Endgame<KPsK>(WHITE),   Endgame<KPsK>(BLACK) };
   Endgame<KPKP>   ScaleKPKP[]   = { Endgame<KPKP>(WHITE),   Endgame<KPKP>(BLACK) };
 
-  // Helper templates used to detect a given material distribution
-  template<Color Us> bool is_KXK(const Position& pos) {
-    const Color Them = (Us == WHITE ? BLACK : WHITE);
-    return  !more_than_one(pos.pieces(Them))
+  // Helper used to detect a given material distribution
+  bool is_KXK(const Position& pos, Color Us) {
```

Comment:
Upper case capital is used for template parameters not for function arguments, please use 'us' instead of 'Us'